.github/actions/binding_setup/action.yml

```diff
@@ -78,19 +78,31 @@ runs:
         r-version: "release"
         use-public-rspm: true
 
-    - name: "Configure ccache for R"
+    - name: Configure ccache for R
       if: inputs.lang == 'R'
       shell: bash
       run: |
-          ccache --set-config "sloppiness=include_file_ctime"
-          ccache --set-config "hash_dir=false"
-          ccache --zero-stats
+          mkdir -p ~/.R
+          echo 'CCACHE=ccache' >> ~/.R/Makevars
+          echo 'CC=$(CCACHE) gcc -std=gnu2x' >> ~/.R/Makevars
+          echo 'CC17=$(CCACHE) gcc -std=gnu17' >> ~/.R/Makevars
+          echo 'CC23=$(CCACHE) gcc -std=gnu2x' >> ~/.R/Makevars
+          echo 'CC90=$(CCACHE) gcc -std=gnu90' >> ~/.R/Makevars
+          echo 'CC99=$(CCACHE) gcc -std=gnu99' >> ~/.R/Makevars
+          echo 'CXX=$(CCACHE) g++' >> ~/.R/Makevars
+          echo 'CXX11=$(CCACHE) g++' >> ~/.R/Makevars
+          echo 'CXX14=$(CCACHE) g++' >> ~/.R/Makevars
+          echo 'CXX17=$(CCACHE) g++' >> ~/.R/Makevars
+          cat ~/.R/Makevars
 
     - name: "Query R dependencies"
       if: inputs.lang == 'R'
       shell: bash
       run: |
           cp src/mlpack/bindings/R/mlpack/DESCRIPTION.in DESCRIPTION
+          echo "R_HOME: $R_HOME"
+          R CMD config CXX
+          R CMD config CXX17
```

Comment:
Could remove those but CI logs are not length constrained so keeping them is fine as well.

Reply:
Ah, you're right, not needed anymore.  Removed.